src/settings-list.js

```diff
@@ -171,8 +171,8 @@ module.exports = [
         toggle: function(value) {
             if (value === true) {
                 $('body').on('dblclick', '.chat-line', function() {
-                    chat.helpers.translate($(this).find('.message'), $(this).data('sender'), $(this).find('.message').data('raw'));
-                    $(this).find('.message').text('Translating..');
+                    chat.helpers.translate($(this).find('.message'), $(this).data('sender'), $(this).find('.message').text());
```

Comment:
I'd rather no longer have `data-raw` be rendered.. afaik this is all it was used for, but if not you need to go back and remove it and update all its dependencies.. The only reason we used .raw was for emote support, but with alt tags it's no longer necessary since .text() can fill those in.